src/Microsoft.DotNet.Interactive/Parsing/DirectiveHelpBuilder.cs

```diff
@@ -14,15 +14,15 @@ public class DirectiveHelpBuilder : HelpBuilder
     private readonly string _rootCommandName;
     private readonly Dictionary<Symbol, string> _directiveHelp = new();
 
-    public DirectiveHelpBuilder(string rootCommandName) : base(LocalizationResources.Instance)
+    public DirectiveHelpBuilder(string rootCommandName) : base(System.CommandLine.LocalizationResources.Instance)
     {
         _rootCommandName = rootCommandName;
     }
 
     public override void Write(HelpContext context)
     {
         using var writer = new StringWriter();
-        new HelpBuilder(LocalizationResources.Instance).Write(context.Command, writer);
+        new HelpBuilder(System.CommandLine.LocalizationResources.Instance).Write(context.Command, writer);
```

Comment:
We should not reference the System.CommandLine loc strings. It's better to have a copy of this loc string in this project, since the System.CommandLine resources won't be public in the next version.

Reply:
The current implementation of HelpBuilder is tightly coupled to System.CommandLine.LocalizationResources. Removing this dependency would require us to create a custom implementation of HelpBuilder, which would involve a significant amount of work.

Given the current state of our project, I propose that we merge the PR as it is. Later, when we plan to remove all dependencies with classes using System.CommandLine, we can allocate time and resources to address this issue properly. This approach will allow us to move forward with the current PR and ensure that we don't introduce any breaking changes or additional complexity at this stage.

Reply:
The System.CommandLine.LocalizationResources API has already been made internal in System.CommandLine and that new version will be pulled into .NET Interactive this week if there's time.

I'd suggest skipping localization for any strings you're pulling from there for now.